.github/workflows/test.yaml

```diff
@@ -214,3 +216,47 @@ jobs:
         run: |
           echo "::error::unexpected status code: got: ${{ steps.lava.outputs.status }}, want: ${{ env.WANT_STATUS }}"
           exit 1
+
+  test-no-vulns:
+    env:
+      WANT_STATUS: 0 # ExitCodeInfo
+    name: Test no vulnerabilities
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      - name: Run Lava Action
+        id: lava
+        uses: ./
+        with:
+          comment-pr: 'true'
+        env:
+          TEST_WORKDIR: .github
```

Comment:
Let's keep it like this for now. But we should create directories under `/testdata` for the different test cases. So, it is more obvious.